### DISCUSSION TITLE
Quad9 DNS

### DISCUSSION THREAD

**Discussion by enessmr** [community/community#138576] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

![{E6AF17D6-02D0-4DCA-81BF-8AE6CB7CC1EA}](https://github.com/user-attachments/assets/f9b01afe-f5ce-44fb-91d0-fe2703061719)
Quad9 DNS. Would it take 24 hours?

**Comment by LiteBrite82** [comment, marked as answer]
I've seen it take up to 48

**Comment by enessmr** [comment]
Thanks

**Reply by Rucellmai61** [reply]
Rucellmai61
